### ISSUE TITLE
[Google Shopping API] All filters options don't appear

### ISSUE THREAD

**Issue by sonika-serpapi** [serpapi/public-roadmap#2043]
A customer reported that only the filter options which are shown on the google shopping page. More options are shown by clicking on "See more", which we don't currently return in the response. 

Currently shown:
<img width="239" alt="image (4)" src="https://github.com/user-attachments/assets/bfbf5f52-157e-4a4f-9dd9-00cabdfa2637">

![image (5)](https://github.com/user-attachments/assets/b00ce1a3-e810-4642-9ced-85cb08fe77e0)

When expanded: [Not currently shown in the response]
<img width="217" alt="image (6)" src="https://github.com/user-attachments/assets/0c33d317-cc6a-446b-9f4d-f42748cfbcd1">

Public links: [Playground](https://serpapi.com/playground?engine=google_shopping&q=patio+forniture+at+Shop.com&location=United+States&gl=us&hl=en&direct_link=true&device=desktop)

Internal links: [Front](https://app.frontapp.com/open/cnv_1ckowvmk?key=AW-GmJjmRlMmBGHe61Yi7Fqfr0lamoRM) | [Search Inspect 1](https://serpapi.com/searches/671183e0c4bcf5f56bf951b1/inspect)

**Comment by schaferyan** [comment]
Thank you @sonika-serpapi. I think this is a `feature request` rather than a `bug`. We only scrape what appears directly on the page by default. If it requires user action (such as clicking "See more") to expand something when viewing the page in the browser, we may or may not be able to support it with SerpApi.

**Comment by sonika-serpapi** [comment]
Sounds good, thank you @schaferyan!

**Comment by Ovi** [comment]
The Google Shopping API indeed makes additional request to retrieve more items, as @schaferyan predicted.

<details>
  <summary><code>https://www.google.com/async/wpr_lhs</code></summary>
  <code>https://www.google.com/async/wpr_lhs?vet=12ahUKEwjdisq_gJiJAxVgk1YBHWAnHEYQ7coMegQIARAD..i&ei=VV4SZ52MOOCm2roP4M7wsAQ&opi=95576897&udm=28&gl=us&hl=en&uule=w%20CAIQICINVW5pdGVkIFN0YXRlcw&yv=3&cs=0&async=query:patio%20furniture%20at%20Shop.com,_basejs:%2Fxjs%2F_%2Fjs%2Fk%3Dxjs.s.en_US.oO8IWFQoyDs.2018.O%2Fam%3DAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAIoIBAAAAAAABQAAQAAAAAAAAAAAAAAAAAAIAAAQAICQEACAAAAACABQCACBAQAAAIAAABAQAAASKAfv85AAAAAAAAAAAAAAQAEwAAAAAA4AIAgAA0BAAAAAgAAMAAAABAAAABAAAABQAAAAIAAAAAAAAAAAAAAAAAACCAANAPAAAAAAAAAAAAAAAAAAAAABqgAIAAfgAAAAAAAQAAAEEAAOCADEAAAAAAAAAA7gOA5wHDIYUFAAAAAAAAAAAAAACAACQI5oD0FwSAAAAAAAAAAAAAAAAAAAAgJWjicgMACQ%2Fdg%3D0%2Fbr%3D1%2Frs%3DACT90oHrP7jB5TjWJpgIgScgKGEpFpconA,_basecss:%2Fxjs%2F_%2Fss%2Fk%3Dxjs.s.aKzrRXdHj4M.L.B1.O%2Fam%3DQOoQIAQAAAQAEdAAAgAAAAAAAAAAAAAAAAAAAAAkAAAABAAAAAAAIAAAIgAAZRIAAJAQAARgAwACAAAfAMABAYABAAAAAAAACQAAAAAAgAAAHQIAAAAAAAAgAAAQQQAAQCEAAAAwEAAAAAMAAAAAIAQIAEAAAyAEAAqQgAA0BKAfhQAAAMAEAABCQBjBMABBBQADOAUAAAAAIAEDAAAQAIQAAACAgAIAIEAAAAiBADAAACQiAAAQABCgCAAEAABgAAAABBAIAICYCeCADEAAAAAAAAAAIAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFAAAAAAAAAAAAAAAAAAAAAAABA%2Fbr%3D1%2Frs%3DACT90oGDyXCgaHuosAR0dl0NaogRpP48-g,_basecomb:%2Fxjs%2F_%2Fjs%2Fk%3Dxjs.s.en_US.oO8IWFQoyDs.2018.O%2Fck%3Dxjs.s.aKzrRXdHj4M.L.B1.O%2Fam%3DQOoQIAQAAAQAEdAAAgAAAAAAAAAAAAAAAAAAAAAkAAAABAAAAAAAIAAIooBAZRIAAJBQAARgAwACAAAfAMABAYABAIAAAQAICQEACAAAgACAHQKACBAQAAAoAAARQQAAQSOAfv85EAAAAAMAAAAAIAQIE0AAAyAE4AqQgAA0BKAfhQgAAMAEAABCQBjBMABBBQADOAcAAAAAIAEDAAAQAIQAACCAgNIPIEAAAAiBADAAACQiAAAQABqgCIAEfgBgAAAABRAIAMGYCeCADEAAAAAAAAAA7gOA5wHDIYUFAAAAAAAAAAAAAACAACQI5oD0FwSAAAAAAAAAAAAAAAAAAAAgJWjicgMACQ%2Fd%3D1%2Fed%3D1%2Fdg%3D0%2Fbr%3D1%2Fujg%3D1%2Frs%3DACT90oF9HnZsb72XYK5sW9VsWLFMszCoAw,_fmt:prog,_id:_VV4SZ52MOOCm2roP4M7wsAQ_28</code>
</details>

**Comment by sonika-serpapi** [comment]
> The Google Shopping API indeed makes additional request to retrieve more items, as @schaferyan predicted.
> 
> `https://www.google.com/async/wpr_lhs`

Thank you for looking into this @Ovi. Are we able to implement this at all? Or will this require an additional parameter or an additional API call or something similar?

**Comment by Ovi** [comment]
> Thank you for looking into this @Ovi. Are we able to implement this at all? Or will this require an additional parameter or an additional API call or something similar?

As far as I know, we currently do not support this.
@aciddjus @zyc9012 @tanys123 can you please answer Sonika's query?

Similar issues: https://github.com/serpapi/public-roadmap/issues/1520, https://github.com/serpapi/public-roadmap/issues/675

**Comment by tanys123** [comment]
Yes, as @Ovi pointed out, it is likely we have to create new engine for it. Let's monitor for some time if more customers request this.

**Comment by AutumroseS** [comment]
Another customer reported this:
[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/215469875172409?view=List)
